### ISSUE TITLE
Is possible to add clients with his IP in a diferent network range than specified LAN

### ISSUE THREAD

**Issue by proura** [brainupdaters/drlm#72]
* Issue type: Discussion/¿bug?
* Impact: Low
* How often it happens: Always
* Brief description of the issue: Listing DRLM clients with "drlm listclient" command I found some clients with wrong IP adress. The IP is not an IP from vlan assigned. Is necessari to show a warning when client is added? Is useful to have clients with IP outside de range? 

##### DRLM Server information:

* DRLM version (/usr/sbin/drlm -V): Disaster Recovery Linux Manager 2.2.0-git / 2017-09-20

**Comment by didacog** [comment]
For me, if its wrong its wrong. having wrong settings could result in networking issues caused by drlm dhcp if not giving correct IP range. So for me not a warning, an error avoiding to have them configured wrong will be better. if the network config is the wrong (eg: bad netmask) then change the network settings(maybe cascade reconf on needed clients) and then let add the client with the correct IP, etc.

**Comment by NefixEstrada** [comment]
Yeah, I also think that DRLM should prevent the user adding clients outside the network. You need to have a valid network to be able to backup / restore clients anyway, so it would be a way to prevent the user from wasting time trying to find why the software isn't working as it should (and maybe opening issues)
